README.md

```diff
@@ -93,3 +93,21 @@ volumes:
 For more information about the different mount configuration options, see the
 [Synced Folders](https://www.vagrantup.com/docs/synced-folders/basic_usage.html) chapter within
 the Vagrant documentation.
+
+### Forwarding ports
+If you want ports from your local machine (the host) forwarded to the vagrant box (the guest), you can specify this in te configuration file.
```

Comment:
There's a small typo here. `te` instead of `the`.

Reply:
Fixed!